quinn-proto/src/connection/mod.rs

```diff
@@ -745,7 +745,7 @@ where
     /// caller to defer materializing a datagram until one can be sent immediately without redundant
     /// checks
     ///
-    /// Returns `Err` iff a `len`-byte datagram cannot currently be sent
+    /// Returns `Err` if a `len`-byte datagram cannot currently be sent
```

Comment:
This isn't a typo. "iff" is an abbreviation for ["if and only if"](https://en.wikipedia.org/wiki/If_and_only_if), which refers to mutual implication.

Reply:
Ah, good to know then.